Tests/Tests/SDAnimatedImageTest.m

```diff
@@ -765,6 +766,15 @@ - (void)test35AnimatedImagePlaybackModeReversedBounce{
     [self waitForExpectationsWithTimeout:15 handler:nil];
 }
 
+- (void)test36AnimatedImageMemoryCost {
+    [[SDImageCodersManager sharedManager] addCoder:[SDImageWebPCoder sharedCoder]];
```

Comment:
Since `SDImageIOCoder` can decode WebP as well on iOS 14. I think we don't need to do anything here.

Reply:
Emm.

Reply:
Use `[[SDImageCodersManager sharedManager] addCoder:[SDImageAWebPCoder sharedCoder]];`
Remember to remove that after test finished. Reset to default state